src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineTextSettingsTab.qml

```diff
@@ -67,13 +67,34 @@ FocusableItem {
 
         SeparatorLine { anchors.margins: -12 }
 
+        SpinBoxPropertyView {
+            id: gapBetweenTextAndLineControl
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.rightMargin: 2
+
+            titleText: qsTrc("inspector", "Gap between text and line")
+            propertyItem: root.model ? root.model.gapBetweenTextAndLine : null
+
+            step: 0.5
+            maxValue: 10.0
```

Comment:
Some last tiny details here: I think users will likely want to make rather fine adjustments to this setting, so a step value of 0.1 might be more convenient. For the maxValue, let's go for 100; that's quite high, but then we can be sure that it will be enough for any purpose. (And we also use 100 as the maximum for the dash/gap lengths for dashed lines in `LineStyleSection.qml`; those settings are maybe somewhat similar.)